TRAMS-API/Models/Request/PostProjectsRequestModel.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace API.Models.Request
+{
+    public class PostProjectsRequestModel
+    {
+        /// <summary>
+        /// The Project Initiator's Full Name. Mandatory.
+        /// </summary>
+        /// <example>Joe Bloggs</example>
+        [Required]
```

Comment:
Warning:  I don't see any checks on field length or pattern so an extra long string could cause an error.  Suggest adding further data annotations for string length and regex patterns where appropriate.  I'd expect that these extra annotations are also reflected automatically in the API documentation if added.

Reply:
Will fix